webcompat/static/js/lib/bugform.js

```diff
@@ -550,7 +558,6 @@ function BugForm() {
             "The image is too big! Please choose something smaller than 4MB.";
           wcEvents.trigger("flash:error", { message: msg, timeout: 5000 });
         }
-        this.loadingIndicator.hide();
```

Comment:
@Regaddi points out that removing this line will fix #2276: https://github.com/webcompat/webcompat.com/issues/2276#issuecomment-375362881

(since loadingIndicator now refers to the form element, we don't want to make it disappear. :))

Reply:
(but we probably still need to replace it with whatever will make the upload loader stop animating)